wiki/Website-Changelog.md

```diff
@@ -0,0 +1,172 @@
+- Use consistent terms for different types of changes: 
+  - Bugfix: “Fixed”
+  - New component, token, variant: “Added”
+  - Update: “Changed”, “Refactored”
+  - Removed or deleted: “Removed”
+  - Other edge cases: “Upgraded”, “Reduced”, “Prevented”
+- When referencing components, use the plain-text component name, e.g., `Accordion`.
+- All components or token names, including those in the summary, should be enclosed in backticks (`).
+
+## Content structure
+
+While best practice guidelines are provided, clarity is most important, so use your best judgment. 
+
+### Component changes
+
+For component changes, place the component name first and enclose it in backticks (`), followed by a dash (-), and then a brief summary.
+
+| Do | Dont |
+|:---|:-----|
+| `Dropdown` - changed the `@height` property to use `max-height` instead of a fixed height.  | Changed the `@height` property in `Dropdown` to use `max-height` instead of a fixed height. |
+
+### New icons
+
+For icons, use a comma-separated list instead of a bulleted list and start with the icon values. Icon names should be enclosed in backticks (`) and use the Ember value.
+
+| Do | Dont |
+|:---|:-----|
+| `twitter-x`, `twitter-x-color`, `aws-cdk`, `aws-cdk-color`, `jfrog`, and `jfrog-color` icons added. | Added a new set of service icons: <ul><li>`twitter-x`</li><li>`twitter-x-color`</li><li>`aws-cdk`</li><li>`aws-cdk-color`</li><li>`jfrog`</li><li>`jfrog-color`</li></ul> | 
+
+### Breaking changes
+
+For breaking changes, add “⛔️ **Breaking change** - ” after the component name/before the summary. Be sure to include relevant details about why this is a breaking change and what needs to be done on the consumer side in order to upgrade to this version.
+
+| Do | Dont |
+|:---|:-----|
+| **`Dropdown ListItem`** - ⛔️ **Breaking change** - Renamed internal CSS classes as follows: <ul><li>from `hds-dropdown-list-item–copy-item` to `hds-dropdown-list-item–variant-copy-item`</li><li>from `hds-dropdown-list-item–description` to `hds-dropdown-list-item–variant-description`</li><li>from `hds-dropdown-list-item–generic` to `hds-dropdown-list-item–variant-generic`</li><li>from `hds-dropdown-list-item–interactive` to `hds-dropdown-list-item–variant-interactive`</li><li>from `hds-dropdown-list-item–separator` to `hds-dropdown-list-item–variant-separator`</li><li>from `hds-dropdown-list-item–title` to `hds-dropdown-list-item–variant-title`</li></ul><br/>_If test assertions are relying on these class names, tests will fail. If extensions/overrides have been applied to these classes, they will suffer visual changes._ | Renamed `Hds::Dropdown::ListItem` internal CSS classes. |
+
+---
+
+## Templates
+
+### Templates for npm packages
+
+Copy and paste this template and adjust as necessary when creating a new [changeset](https://github.com/hashicorp/design-system#changesets). _Replace all elements in brackets._
+
+#### Simple entry
+```
+**`{component-name}`** - Fixed {...additional details}.
```

Comment:
Ok, here's is bold again. Now I am not sure anymore which one you prefer. Do you want me to do a test for you where I show you the before and after?